Makefile

```diff
@@ -669,6 +669,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: kind-reset
+kind-reset: ## Destroys the "capi-test" kind cluster.
+	kind delete cluster --name=capi-test || true
+
+.PHONY: tilt-up
+tilt-up: kind-cluster ## Start tilt and build kind cluster if needed.
```

Comment:
I frequently stop tilt locally without deleting the cluster and do `tilt up`. It would be nice if the `make tilt-up` command is re-entrant as I would personally not want to switch between doing `make tilt-up` the first time and `tilt up` the subsequent times.